README.asciidoc

```diff
@@ -131,6 +134,13 @@ The plugin defines the following extension properties in the `docker` closure:
 |`apiVersion`    |String    |null                     |The remote API version. For most cases this can be left null.
 |=======
 
+[WARNING]
+====
+Default value of `docker.url` property is deprecated.
+It will be replaced to `unix:///var/run/docker.sock` in upcoming releases.
+Please adjust your build scripts accordingly.
```

Comment:
You've written
>>That's assuming you're connecting to a local engine. You may not have docker installed at all locally and just want to connect to some remote instance to do work (we do that a lot here for CI related purposes).

If default will be `tcp://localhost:2375` - it will not work, because `Docker Engine` runs on other PC and is not accesible via `localhost` URL.